src/pages/Profile.jsx

```diff
@@ -9,16 +9,64 @@ import {
 import { useLocation, useNavigate } from "react-router-dom";
 
 import { FaUserCircle } from "react-icons/fa";
+import { useState } from "react";
+import {auth,storage,} from "../lib/firebase";
+import { useSnackbar } from "notistack";
 
 function Profile() {
   const { name, email, avatar } = useLocation().state;
   const isNonMobile = useMediaQuery("(min-width: 768px)");
+  const { enqueueSnackbar } = useSnackbar();
+  const [image, setImage] = useState("");
+  const [img, setAvatar] = useState(avatar);
```

Comment:
usestate should be declare as avatar,setAvatar or img,setImg

Reply:
or profilepic or anything meaningful

Reply:
changed